nuxeo-runtime/nuxeo-connect-standalone/src/test/java/org/nuxeo/launcher/connect/TestConnectBroker.java

```diff
@@ -76,6 +79,10 @@
 
     final File nuxeoHome = new File("target/launcher");
 
+    public static final PackageState NON_EXISTENT = null;
```

Comment:
I agree with you. I don't like to use this shortcut neither; by using this shortcut, I simply want to:

- Check a package is non-existent
- Align with the package-status assertion on other packages using `#assertPkgs` or `#checkPackagesState`; and not to use `assertNull()` which has another style

We should have a better class / method to provide the package status for non-existent ones in the source code `src`, it should not be responsibility of the `test` code to do so.